test/__snapshots__/index.spec.js.snap

```diff
@@ -397,6 +397,58 @@ Array [
 ]
 `;
 
+exports[`should pass on fixtures/rules/css-modules/undefined.js 1`] = `
+Array [
+  Object {
+    "column": 41,
+    "line": 8,
+    "rule": "css-modules/no-undef-class",
+    "severity": 2,
+  },
+]
+`;
+
+exports[`should pass on fixtures/rules/css-modules/unused.js 1`] = `
+Array [
+  Object {
+    "column": 8,
+    "line": 2,
+    "rule": "css-modules/no-unused-class",
+    "severity": 1,
+  },
+  Object {
+    "column": 47,
+    "line": 9,
+    "rule": "react/jsx-curly-spacing",
```

Comment:
These errors shouldn't be here because you have some jsx linting errors. See my suggestions below.